homeassistant/components/sensor/postnl.py

```diff
@@ -0,0 +1,128 @@
+"""
+Sensor for PostNL packages.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.postnl/
+"""
+import logging
+from datetime import timedelta, datetime
+import re
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (CONF_NAME, CONF_USERNAME, CONF_PASSWORD,
+                                 ATTR_ATTRIBUTION)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['postnl_api==0.3']
+
+_LOGGER = logging.getLogger(__name__)
+CONF_UPDATE_INTERVAL = 'update_interval'
+DEFAULT_NAME = 'postnl'
+ICON = 'mdi:package-variant-closed'
+ATTRIBUTION = 'Information provided by PostNL'
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_USERNAME): cv.string,
+    vol.Required(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_UPDATE_INTERVAL, default=timedelta(seconds=1800)):
```

Comment:
No need to implement this.

Implement the default as a constant SCAN_INTERVAL. Home Assistant will be able to pick that up and users can override it via their config: https://home-assistant.io/docs/configuration/platform_options/#scan-interval